StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerBodyView.swift

```diff
@@ -75,8 +60,9 @@ final class LinkAccountPickerBodyView: UIView {
                 }(),
                 balanceString:
                     (accountTuple.accountPickerAccount.caption == nil) ? rowTitles.balanceString : nil,
-                isSelected: selectedAccountTuple?.partnerAccount.id == accountTuple.partnerAccount.id
+                isSelected: false // initially nothing is selected
```

Comment:
one of the core insights on analyzing the correctness of the code is that the _only_ thing that changed  each time `selectAccount` was fired is `isSelected` (the rows never changed because they are based off the instantiated instance variables)